unity-renderer/Assets/DCLServices/SubscriptionsAPIService/SubscriptionsAPIService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Cysharp.Threading.Tasks;
+using DCL;
+using System.Threading;
+
+namespace DCLServices.SubscriptionsAPIService
+{
+    public interface ISubscriptionsAPIService : IService
+    {
+        UniTask<Subscription> CreateSubscription(string email, CancellationToken ct);
+        UniTask DeleteSubscription(string subscriptionId, CancellationToken ct);
+        UniTask<Subscription> GetSubscription(string subscriptionId, CancellationToken ct);
+    }
+
+    public class SubscriptionsAPIService : ISubscriptionsAPIService
+    {
+        private readonly ISubscriptionsAPIClient client;
+
+        public SubscriptionsAPIService(ISubscriptionsAPIClient client) =>
+            this.client = client;
```

Comment:
What's the point of wrapping up `ISubscriptionsAPIClient` with this class?
SubscriptionsAPIService implements ISubscriptionsAPIService which can be substituted by anything in the future

Reply:
Yep, I think you're right... I wanted to follow the same architecture that we have for the Places API and, now that I'm seeing it, it's also having the same unneeded abstraction.

If you don't mind, in order to ease the current feature's release, I'm going to annotate it and I will apply the change during the hardening week 👍

Thanks!